test/NetCoreTool.Template.WebApi.Test/Models/TemplateOptions.cs

```diff
@@ -14,10 +14,6 @@ public IEnumerator<object[]> GetEnumerator()
                 {
                     new object[] { "3.2.0-rc1", "net6.0", "C#", },
                     new object[] { "3.2.0-rc1", "net6.0", "F#", },
```

Comment:
I just noticed the -rc1 here, but I'm not exactly sure what this does... should it be 3.2.5 to match our latest patch release?

Reply:
yes, will address that in separate PR